src/main/resources/i18n/servlets/challenges/sqli/sqli5.properties

```diff
@@ -3,6 +3,7 @@ response.orderComplete.p1 = Your order has been made and has been sent to our ma
 response.orderComplete.p2 = Your order comes to a total of 
 response.trollsFreeSolution = Trolls were free, Well Done - 
 response.orderFailed = Order Failed - Please try again later
+response.invalidAmount = Order Failed - Ordered amounts must be between 0 and 9000

```

Comment:
Clear lack of DBZ reference. Sure you don't want to put it in before I merge?

Reply:
How could I say no to that? I've updated the error based off what the scanner says about its power level.